doc/developer/building-frr-for-centos7.rst

```diff
@@ -14,14 +14,21 @@ CentOS 7 restrictions:
 Install required packages
 -------------------------
 
+Add EPEL Repository:
+
+::
+
+    sudo yum -y install epel-release
+
 Add packages:
 
 ::
 
     sudo yum install git autoconf automake libtool make \
       readline-devel texinfo net-snmp-devel groff pkgconfig \
       json-c-devel pam-devel bison flex pytest c-ares-devel \
-      python-devel systemd-devel python-sphinx libcap-devel
+      python-devel systemd-devel python-sphinx libcap-devel \
+      python3-sphinx
 
 .. include:: building-libyang.rst
```

Comment:
please add:
cmake rpm-build  python3-devel

Reply:
Fixing it. And removing python-devel and python-sphinx (as we use the python3 tools).
Will update as soon as a testbuild without them is done

Reply:
rpm-build is part of the package build and documented there (not used for just a normal build).
Add the others and removed python-devel and python-sphinx (which are python2 based)